spec/models/state_file_az_intake_spec.rb

```diff
@@ -77,7 +77,6 @@
 #
 # Indexes
 #
-#  index_state_file_az_intakes_on_email_address        (email_address)
```

Comment:
Weird - I think this may be related to a weird annotation glitch Mike was working on the other week.